packages/apps/plugins/plugin-metadata/.storybook/main.cjs

```diff
@@ -0,0 +1,50 @@
+const { resolve } = require('path');
```

Comment:
i've started to clean these up and make them the same (which includes the copyright header). a lot of the current storybooks don't include the theme.